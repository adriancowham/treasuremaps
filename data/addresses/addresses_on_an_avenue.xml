<posts>
<item rdf:about="http://sacramento.craigslist.org/gms/2339976701.html"> 
<title><![CDATA[Yard Sale Bird & Squirrel Feeders (fair oaks) $1]]></title> 
<link>http://sacramento.craigslist.org/gms/2339976701.html</link> 
<description><![CDATA[SQUIRREL & BIRD FEEDERS
<br> 
QUILTS, PUIRSES,  FLOWER POTS, TV CABINET, 
<br> 
MUCH MORE MUST COME AND TAKE A PEEK.........
<br> 
8915 VINCENT AVE. FAIR OAKS, CA 95628
<br> 
SATURDAY 4-23-11 8:00-12:00<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=fair oaks -->Location: fair oaks
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T22:50:57-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339976701.html</dc:source> 
<dc:title><![CDATA[Yard Sale Bird & Squirrel Feeders (fair oaks) $1]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T22:50:57-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2339972144.html"> 
<title><![CDATA[Yard Sale in Fair Oaks/ 7am-Noon (6725 Admiral Avenue, Fair Oaks)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339972144.html</link> 
<description><![CDATA[Yard sale with lots of smalls.  Old Pyrex cookware,  Vintage Ceiling Fan, Old Windows, Lots of Sizes. <!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=6725 Admiral Avenue, Fair Oaks -->Location: 6725 Admiral Avenue, Fair Oaks
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T22:39:40-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339972144.html</dc:source> 
<dc:title><![CDATA[Yard Sale in Fair Oaks/ 7am-Noon (6725 Admiral Avenue, Fair Oaks)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T22:39:40-07:00</dcterms:issued> 
</item> 

</posts>