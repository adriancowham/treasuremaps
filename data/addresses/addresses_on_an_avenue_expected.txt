8915 VINCENT AVE.
6725 Admiral Avenue